crates/polars-pipe/src/pipeline/mod.rs

```diff
@@ -28,6 +28,8 @@ pub(crate) fn determine_chunk_size(n_cols: usize, n_threads: usize) -> PolarsRes
         val.parse().map_err(
             |_| polars_err!(ComputeError: "could not parse 'POLARS_STREAMING_CHUNK_SIZE' env var"),
         )
+    } else if n_cols == 0 {
+        Ok(1)
```

Comment:
I'm a bit unsure which `chunk_size` are suitable here, not choose `0` is to avoid introducing new division by 0 exception.